esp-hal/src/soc/psram_common.rs

```diff
@@ -23,3 +25,16 @@ impl PsramSize {
         matches!(self, PsramSize::AutoDetect)
     }
 }
+
+/// Returns the address range available in external memory.
+#[cfg(any(feature = "quad-psram", feature = "octal-psram"))]
+pub fn psram_range(_psram: &crate::peripherals::PSRAM) -> Range<usize> {
```

Comment:
Is there much point in taking `&crate::peripherals::PSRAM`?

Reply:
Yes, users are forced to call esp_hal::init before